src/sentry/tasks/merge.py

```diff
@@ -257,11 +257,16 @@ def merge_objects(models, group, new_group, limit=1000,
                             )
                     elif model == GroupTagValue:
                         with transaction.atomic(using=router.db_for_write(model)):
```

Comment:
im not sure either @tkaemming, but its a pita to read code on github PRs :(:(:(:(:(

its possible the code used to do more than this?